content/main/contrib_help.md

```diff
@@ -0,0 +1,28 @@
+# Getting started as a DlangScience contributor
+
+Here is a pre-alpha draft of a bit of "getting started" documentation...
+
+
+## Contributing to DlangScience
+
+All the packages which make up DlangScience are hosted in their own github projects at https://github.com/DlangScience/package. For the time being, development is proceeding quite independently in the projects. For example, issues are tracked separately within the respective Github site and each project can be forked and cloned separately. The mild exception to this concerns user and API documentation, as outlined in the documentation section below.
```

Comment:
Please change https://github.com/DlangScience/package to ```https://github.com/DlangScience/<packageName>```, in verbatim/code (i.e. triple backticks)

> within the respective Github site

should be "within their respective Github sites"

> The mild exception to this concerns user and API documentation, as outlined in the documentation section below.

I don't understand. user and API documentation for each project is held within each independent repository.